### PR TITLE
Home navbar (Tabs and Routing)

### PR DESCRIPTION
**Pull request by Disquietude**
The home navbar with working tabs and routing between pages (profile and logout still in progress). Looking to merge in early so that @airensurzyn  and @umbauk have access to the routing functionality while they're working on the front-end.

### CODE REVIEW

File: client/src/Components/Styles/Colors.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const colors = {
```
Comment (airensurzyn) [inline client/src/Components/Styles/Colors.js:1]: Nice, very convenient

**Review by umbauk — APPROVED** [review]
Looks great! Just some general UI feedback.

I noticed it's not responsive so you may want to add a card for that so we don't forget. While a user is unlikely to set up campaigns on mobile, I imagine a common user use case would be checking the status of campaigns on mobile.

Should we also have the user name in top right as per mocks? It could then show "Log Out" when you click. Would make some testing easier if we had ability to log out in the app too :) That may be a lot of work though and require a new card.

The user will expect clicking the logo to go to the home page. Maybe it should be a link to the Campaigns page?

**Review by airensurzyn — APPROVED** [review]
Looks good to me!

**Comment by Disquietude** [comment]
> Looks great! Just some general UI feedback.
> 
> I noticed it's not responsive so you may want to add a card for that so we don't forget. While a user is unlikely to set up campaigns on mobile, I imagine a common user use case would be checking the status of campaigns on mobile.
> 
> Should we also have the user name in top right as per mocks? It could then show "Log Out" when you click. Would make some testing easier if we had ability to log out in the app too :) That may be a lot of work though and require a new card.
> 
> The user will expect clicking the logo to go to the home page. Maybe it should be a link to the Campaigns page?

Thanks @umbauk ! You are absolutely right about the responsiveness, although I imagine in a mobile setting we would use a drawer or something rather than a navbar, which would require building a separate component. I'll create an issue for responsive components for the icebox.

As for the username/logout thing, I'm working on that right now! It'll be in the next PR for the navbar. I just want to merge this part early so you'll have access to it when you're working on your front-end pages.

**Pull request merged**